app/src/main/java/com/example/quizacademytask/StackListFragment.kt

```diff
@@ -67,7 +70,13 @@ class StackListFragment : Fragment(), SimpleAdapter.OnItemClickListener {
         courseJSON = ""
         swipeContainer = binding.swipeContainer
         idlingResource = CountingIdlingResource("API")
+
+        //Set the toolbar
         binding.toolbarStackList.title = "Topics"
+        (activity as MainActivity).setSupportActionBar(toolbar)
```

Comment:
does it have to be specifically `MainActivity` to be able to setSupportActionBar?